### ISSUE TITLE
switch unit tests to using coverage-instrumented cc-runtime binary

### ISSUE THREAD

**Issue by jodh-intel** [clearcontainers/runtime#359]
@dlespiau worked out how to create a coverage-instrumented `cc-runtime` binary (#186).

We now need to switch to using this since the Coveralls figures only show the unit test coverage - the figure does not include the additional coverage gained by running the integration and system tests.

To do this, we need to switch the CI test scripts to:

- run all the tests using the correct coverage-instrumented binary under SemaphoreCI (not Travis).
- append the additional results to the coverage file
- upload the results to Coveralls.

**Comment by jodh-intel** [comment]
Related: #93.

**Comment by egernst** [comment]
@jodh-intel can this be closed at this point?

**Comment by jodh-intel** [comment]
I don't think it matters where it is run per se, just that we do it. I agree that this is post-release (tagged).
